mkdocs.yml

```diff
@@ -44,6 +44,8 @@ markdown_extensions:
   - pymdownx.highlight
   - footnotes
   - def_list
+  - attr_list
+  - md_in_html
```

Comment:
I don't think the `md_in_html` extension is being used here, only the `attr_list` one is necessary for those images. 
```suggestion
```